SPECS/fontconfig/fontconfig.spec

```diff
@@ -1,8 +1,8 @@
 Summary:        library for configuring and customizing font access.
 Name:           fontconfig
-Version:        2.13.91
+Version:        2.13.95
 Release:        1%{?dist}
-License:        BSD/GPL
+License:        MIT/Unicode/Public Domain
```

Comment:
Please change the license line to separate licenses with `and` and `or`, so the overall licensing is clearer.